### PR TITLE
Export TwilioClient in typescript definitions

### LINKED ISSUE #387: Typing via index.d.ts is incomplete
Typing does not export the type of the client (`TwilioClient`)

```ts
    import * as twilio from 'twilio';
    private client: twilio: any; //type is not exported

    constructor(config: TwilioConfig) {
        this.client = twilio(config.account_sid, config.auth_token);
    }

```

### PR DESCRIPTION
**Pull request by ewisuri**
Resolves #387

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -16,10 +16,11 @@ interface TwimlConstructor<T> {
 declare function twilio(
   accountSid?: string,
   authToken?: string,
-  opts?: TwilioClient.TwilioClientOptions
-): TwilioClient;
+  opts?: _TwilioClient.TwilioClientOptions
+): _TwilioClient;
 
 declare namespace twilio {
+  export import TwilioClient = _TwilioClient;
```
Comment (dkundel) [inline index.d.ts]: In `lib/index.js` this is being exported as `Twilio` and not `TwilioClient`. Could you update this please?

**Comment by dkundel** [comment]
Hey @ewisuri!

Thank you so much for your contribution! I took a look at it and it's

File: index.d.ts (hunk 1 above)
Comment (ewisuri) [reply index.d.ts]: Updated. Also reverted the '_' prefixing since renaming to Twilio removes the conflict with the local TwilioClient

**Comment by dkundel** [comment]
This looks good on me, but just to make sure @cjcodes can you merge this in :)

Thanks,
Dominik

**Comment by cjcodes** [comment]
Away it goes! This will get into the next release.

**Pull request merged**